cockroachcloud/migrations-page.md

```diff
@@ -83,23 +78,87 @@ The dump file must be smaller than 4 MB. `INSERT` and `COPY` statements will be
 </section>
 </ul>
 
-<li>Click <b>Migrate</b> and wait for the schema to be analyzed. A loading screen is displayed. Depending on the size and complexity of the SQL dump, analyzing the schema can require up to several minutes.</li>
-<li>When analysis is complete, review the <a href="#summary-report"><b>Summary Report</b></a> and edit, add, or remove SQL statements in the <a href="#statements-list"><b>Statements</b> list</a>.</li>
+</li>
+<li>Click <b>Next</b>.</li>
+<section class="filter-content" markdown="1" data-scope="postgres mysql">
+<li>Use either the <a href="#upload-file"><b>Upload File</b></a> or <a href="#use-credentials"><b>Use Credentials</b></a> option to add your schema.
+</section>
 </ol>
 
-## Migrations table
+#### Upload File
```

Comment:
I think we should put the credentials workflow first if possible because it's easier